parts/k8s/addons/coredns.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
     rbac.authorization.kubernetes.io/autoupdate: "true"
   labels:
     kubernetes.io/bootstrapping: rbac-defaults
-    addonmanager.kubernetes.io/mode: EnsureExists
```

Comment:
So, same comment about `ConfigMap` resources everywhere, but I'm torn on this particular `ClusterRoleBinding` as it is set to `EnsureExists` in the reference yaml:

https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/coredns/coredns.yaml.in

But I don't know why you wouldn't want to set this to `Reconcile` to allow the spec to adapt over time. @lachie83 @khenidak Any idea why you would bootstrap an addon spec w/ a `ClusterRoleBinding` set to `EnsureExists`? (That means essentially that the next time the addon manager picks up a change to this resource from `/etc/kubernetes/addons/*` it will not automatically reconcile it in the runtime)

Reply:
I left this as `EnsureExists` for now